test/util.ts

```diff
@@ -30,6 +30,8 @@ export const platform = mocked(_platform);
 export const env = mocked(_env);
 export const hostRules = mocked(_hostRules);
 
+Object.assign(platform, { commitFiles: jest.fn() });
+
```

Comment:
If we do this, then it should be done in `util/global`, where we init the mocked platform?

Reply:
What file is that? This is the file where we mock and export platform, so I thought to add it here.